### PR TITLE
Healthcheck endpoint

### PR DESCRIPTION
**Pull request by cv65kr**
## What was changed
It was add a new endpoint for health checking ui-server.

## Why?
It's very useful for AWS ALB

### CODE REVIEW

**Comment by feedmeapples** [comment]
thx for the PR

could you put this route in a server/routes/health.go and follow the same pattern as the other routes are set

File: server/server.go [hunk 1]
```diff
@@ -105,6 +105,10 @@ func NewServer(opts ...server_options.ServerOption) *Server {
 		panic(err)
 	}
 
+	e.GET("/healthz", func(c echo.Context) error {
```
Comment (feedmeapples) [inline server/server.go:108]: could you put this route in a server/routes/health.go and follow the same pattern as the other routes are set

**Review by feedmeapples — CHANGES_REQUESTED** [review]

**Comment by feedmeapples** [comment]
thx for the PR

File: server/server.go (hunk 1 above)
Comment (cv65kr) [reply server/server.go]: Done :)

Comment (feedmeapples) [reply server/server.go]: makes sense, still want to make the `/route` folder help with the high level picture of the available routes

**Review by feedmeapples — APPROVED** [review]

**Comment by feedmeapples** [comment]
e2e tests fail due to security restrictions for external contributors

**Pull request merged**